### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by pjmattingly** [serverless/serverless#11195] [General]
Hello,

As the contribution guide said I should start by reaching out the community, I wanted to introduce myself. I've been developing in academia for a lot of years, but I want to move toward the private sector. And so my motivation is get my feet wet with a larger project with other experienced developers to get my feet wet.

Cheers.

--Peter

**Comment by pgrzesik** [comment]
Hey @pjmattingly and welcome to the Serverless Framework community 🙌 We'd be happy to accept contributions and if you have any questions, feel free to reach out - I'll try to help.
